content/en/docs/concepts/services-networking/service.md

```diff
@@ -699,7 +699,7 @@ metadata:
 metadata:
     name: my-service
     annotations:
-        service.beta.kubernetes.io/aws-load-balancer-internal: "true"
```

Comment:
According to https://cloud-provider-aws.sigs.k8s.io/service_controller/, `service.beta.kubernetes.io/aws-load-balancer-internal: "true"` is still the right thing to se.

We could mention `service.beta.kubernetes.io/aws-load-balancer-scheme: "internal"` as well, because the AWS load balancer controller is part of Kubernetes (AWS donated the code).

Reply:
Thanks, I raised an issue (https://github.com/kubernetes/cloud-provider-aws/issues/624) to see if we should deprecate `service.beta.kubernetes.io/aws-load-balancer-internal: "true"` annotation :)